lgc/patch/Gfx6ConfigBuilder.cpp

```diff
@@ -890,9 +890,38 @@ void ConfigBuilder::buildPsRegConfig(ShaderStage shaderStage, T *pConfig) {
   SET_REG_FIELD(&pConfig->psRegs, PA_SC_MODE_CNTL_1, FORCE_EOV_CNTDWN_ENABLE, true);
   SET_REG_FIELD(&pConfig->psRegs, PA_SC_MODE_CNTL_1, FORCE_EOV_REZ_ENABLE, true);
 
+  bool enableEarlyTests = fragmentMode.earlyFragmentTests;
```

Comment:
I looks you only check stencil test status, without check whether depth/stencil is modified in shader.
does it limited by spec?

Reply:
Depth state was implemented with register control. it is different from stencil. you can see the register "DB_SHADER_CONTROL"